src/wallet/utils/validChainId.ts

```diff
@@ -0,0 +1,15 @@
+import { validChainIdOptions, validChainIdResponse } from '../types';
+import { baseSepolia, base } from 'viem/chains';
+
+/**
+ * validChainId
+ *  - Checks if the paymaster operations chain id is valid
+ *  - Only allows the base and baseSepolia chain ids
+ */
+export function validChainId({ chainId }: validChainIdOptions): validChainIdResponse {
```

Comment:
Curious about your thoughts of naming convention. 

Planning for `validChainId`, `validEntryPoint`, etc. 

We might want to add `sponsor` or `paymaster` such a `validSponsoredChainId`, `validSponsoredEntryPoint`, etc.

Reply:
If something returns "something" a verb is usually a best practice, so in this case I will just call it `isBase`.